lib/create-config-gypi.js

```diff
@@ -75,6 +75,9 @@ async function getCurrentConfigGypi ({ gyp, nodeDir, vsInfo }) {
   // set the node development directory
   variables.nodedir = nodeDir
 
+  // set the configured Python path
+  variables.python = python
```

Comment:
My only concern is, if there are any existing packages which end up setting a variable `variables.python` in their `confi.gypi`?

Maybe this should be something more specific where you'd really have to try to collide with our namespace, like `variables.node_gyp.python_path` or `variables.node_gyp_python_path`?

(I'm admittedly not a heavy `node-gyp` or `gyp` user, so maybe someone would know if that's a realistic concern or not?)